stubs/requests/requests/help.pyi

```diff
@@ -0,0 +1,35 @@
+from typing import TypedDict
```

Comment:
```suggestion
from typing_extensions import TypedDict
```

Reply:
Thanks — was just moving back to this after adding the test case for weakref :)